tests/tests.js

```diff
@@ -59,6 +59,21 @@ describe("htmlbars-inline-precompile", function() {
     }, /placeholders inside a tagged template string are not supported/);
   });
 
+  it("moduleName is supplies to the precompiler", function() {
+    var fullpath = path.join(__dirname, 'fixtures', 'hello-world.js');
+
+    var precompile = function(template, options) {
+      assert.equal(options.moduleName, fullpath);
```

Comment:
I have just one more question: I have never used the `moduleName` option so forgive my ignorance, but since we're using `relativeName`, wouldn't we expect the `moduleName` be a relative path here? I have seen that there is the `sourceRoot` option for `babel.transformFileSync`, which might be the one to specify to which directory the file should be resolved relative to?

I am just asking. You said you used that successfully in one of your plugins and it works, so I am just asking because I want to understand what the `moduleName` option is used for...